src/checkstyle/checks/whitespace/LineCheckBase.hx

```diff
@@ -0,0 +1,34 @@
+package checkstyle.checks.whitespace;
+
+@ignore("base class line based whitespace checks")
+class LineCheckBase extends Check {
+
+	var quotesRE:EReg;
+	var multilineStartRE:EReg;
+	var escapeRE:EReg;
+	var multilineStringStart:Bool;
+
+	public function new() {
+		super(LINE);
+		quotesRE = ~/('|")/;
+		multilineStartRE = null;
+		escapeRE = null;
+		multilineStringStart = false;
+	}
+
+	function isMultineString(line:String):Bool {
```

Comment:
How can you reliably tell whether something is within a _multiline_ string based on a _single_ line?